orchagent/portsorch.cpp

```diff
@@ -752,62 +752,200 @@ bool PortsOrch::setPortPfcAsym(Port &port, string pfc_asym)
     return true;
 }
 
-bool PortsOrch::createBindAclTableGroup(sai_object_id_t id, sai_object_id_t &group_oid, acl_stage_type_t acl_stage)
```

Comment:
Please add description about the refactor in this file.

Reply:
I feel, code is self explanatory, I have moved repeated lines to common function!
Other changes are related to ACL dependency handling new code.

Reply:
Sure, you can just mention this in the PR description or commit message, no need in the code.